src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -271,6 +274,18 @@ namespace winrt::TerminalApp::implementation
 
         _isAlwaysOnTop = _settings.GlobalSettings().AlwaysOnTop();
 
+        // This lambda is a local because there will be more toasts in the
+        // future that will need the same logic.
+        auto safeRefocus = [weakThis{ get_weak() }](auto&&, auto&&) {
+            if (auto page{ weakThis.get() })
+            {
+                page->_FocusActiveControl(nullptr, nullptr);
+            }
+        };
+
+        _windowIdToast = std::make_unique<Toast>(WindowIdToast());
```

Comment:
using a unique_ptr for this scares me

also, should we really force the toast to be loaded here? it's a whole UI element that we won't need in, like, 99% of cases. If we make it x:Load and promote safeRefocus to a function, we can let xaml hand binding it when the control loads.

Reply:
(I'd like our startup to be as light as possible)

Reply:
Sure okay